### PR TITLE
Count function

### PR DESCRIPTION
**Pull request by Sud1691**
updated main.tf file with count paramter.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -70,84 +63,50 @@ resource "aws_route_table" "publicRouteTableA" {
...
-  tags = {
-    Name = var.privateRouteTable[1]
-  }
-}
-
-resource "aws_route_table_association" "PrivateAssociationA" {
-  subnet_id      = aws_subnet.privateSubnetA.id
-  route_table_id = aws_route_table.privateRouteTableA.id
+resource "aws_route_table_association" "PrivateAssociation" {
+  count = 2
```
Comment (geoffrey1330) [inline main.tf:87]: could you please change the hardcoded value 2 to variable length e.g length(var.Public_subnet)

File: main.tf [hunk 2]
```diff
@@ -70,84 +63,50 @@ resource "aws_route_table" "publicRouteTableA" {
...
-resource "aws_route_table_association" "PublicAssociationA" {
-  subnet_id      = aws_subnet.publicSubnetA.id
-  route_table_id = aws_route_table.publicRouteTableA.id
-}
-
-resource "aws_route_table_association" "PublicAssociationB" {
-  subnet_id      = aws_subnet.publicSubnetB.id
-  route_table_id = aws_route_table.publicRouteTableA.id
+resource "aws_route_table_association" "PublicAssociation" {
+  count = 2
```
Comment (geoffrey1330) [inline main.tf:67]: could you please change the hardcoded value 2 to variable length e.g length(var.Public_subnet)

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (Sud1691) [reply main.tf]: Updated count parameter. Thanks for highlighting.

File: main.tf (hunk 2 above)
Comment (Sud1691) [reply main.tf]: Updated count parameter. Thanks for highlighting.

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**